edc-extensions/dataplane/dataplane-self-registration/build.gradle.kts

```diff
@@ -21,6 +21,8 @@ dependencies {
     implementation(libs.edc.spi.web)
     implementation(libs.edc.spi.dataplane.selector)
 
+    runtimeOnly(project(":edc-extensions:dataplane:pipeline-service"))
+
```

Comment:
I would not put the dependency here, but in the `edc-dataplane-base`

Reply:
you're right, thank you, updated in https://github.com/eclipse-tractusx/tractusx-edc/pull/1520/commits/d72b9a2b574f7d97cf2556e4de99c6be8e62b48d!